package.json

```diff
@@ -26,7 +26,7 @@
     "install:wasi": "bash scripts/setup_wasi.sh",
     "install:wing-api-checker": "npm --prefix apps/wing-api-checker --install-links=false install",
     "install:wing-playground": "npm --prefix apps/wing-playground --install-links=false install",
-    "install:winglang": "npm --prefix apps/wing --install-links=false install",
+    "install:winglang": "npm --prefix apps/wing --install-links=false install --ignore-scripts",
```

Comment:
What does this change do? (just curious for learning)

Reply:
I'm trying to find a way around this, but a preinstall script was causing an error: https://github.com/winglang/wing/actions/runs/4206774554/jobs/7300707121#step:6:213

winglang currently doesn't have any other install hooks, so this isn't a complete deal breaker but I'll see if there's some other way to fix it.

Reply:
Caused by https://github.com/pnpm/only-allow/issues/13

Keeping this changed in though, because ignoring scripts doesn't hurt us atm